docs/security/clerk-csp.mdx

```diff
@@ -41,4 +44,6 @@ const nextConfig = {
 };
 
 module.exports = nextConfig;
-```
\ No newline at end of file
+```
+
+Note that you will need to make sure any third-party domains where scripts or assets are loaded from are also specified.
```

Comment:
Do we have a component to make the  note more noticable?

Reply:
Yeah, can update!